Docs/source/pages/commands.md

```diff
@@ -6,15 +6,17 @@ If you are looking for information on a specific function, class or method, this
 
 The **AltDriver** class represents the main app driver component. When you instantiate an AltDriver in your tests, you can use it to "drive" your app like one of your users would, by interacting with all the app objects, their properties and methods.
 
-An AltDriver instance will connect to the running instrumented Unity application. In the constructor, we need to tell the driver where (on what IP and on what port) the instrumented Unity App is running. We can also set some more advanced parameters, as shown in the table below:
+An AltDriver instance will connect to the running instrumented Unity application. In the constructor, we need to tell the driver where (on what IP and on what port) the instrumented Unity App with a specific name is running and for how many seconds to let the communication opened. We can also set some more advanced parameters, as shown in the table below:
```

Comment:
The last sentence seems unnecessary as all parameters are already mentioned before, therefore no other advanced parameters remain to be described.